war/src/main/java/io/lumeer/engine/rest/dao/ViewDao.java

```diff
@@ -28,6 +33,9 @@
  */
 public class ViewDao {
 
+   @Inject
+   private UserFacade userFacade;
```

Comment:
This must not be here! While this class is badly named, it was meant to be a [DTO](https://en.wikipedia.org/wiki/Data_transfer_object). It should only carry data  and if such information as currently logged user is required, it needs to be passed to it as a constructor argument when it is created.